packages/shared/src/components/sidebar/ContributeSection.tsx

```diff
@@ -0,0 +1,86 @@
+import dynamic from 'next/dynamic';
+import React, { ReactElement, useContext, useState } from 'react';
+import AnalyticsContext from '../../contexts/AnalyticsContext';
+import FeaturesContext from '../../contexts/FeaturesContext';
+import EmbedIcon from '../icons/Embed';
+import LinkIcon from '../icons/Link';
+import { ListIcon, SidebarMenuItem } from './common';
+import { Section, SectionCommonProps } from './Section';
+
+const SubmitArticleModal = dynamic(
+  () =>
+    import(
+      /* webpackChunkName: "SubmitArticleModal" */ '../modals/SubmitArticleModal'
+    ),
+);
+
+const NewSourceModal = dynamic(
+  () =>
+    import(/* webpackChunkName: "NewSourceModal" */ '../modals/NewSourceModal'),
+);
+
+export function ContributeSection(props: SectionCommonProps): ReactElement {
+  const {
+    canSubmitArticle,
+    submitArticleOn,
+    submitArticleSidebarButton,
+    submitArticleModalButton,
+  } = useContext(FeaturesContext);
+  const { trackEvent } = useContext(AnalyticsContext);
```

Comment:
Same with this AnalyticsContext, I see some components use it, but they all part of the sidebar anyway.
So maybe we can simply pass it down?